engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -65,11 +65,11 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
             var success, error, params = getParams(), removalPromise;
 
             success = function (response) {
-                var message = translate('Removal of selected repositories initiated successfully. ');
-
-                message += translate('<a href="/foreman_tasks/tasks/%taskId">Click here to check the status of the task.</a>').
-                    replace('%taskId', response.task.id);
-                Notification.setSuccessMessage(message);
+                var message = translate('Removal of selected repositories initiated successfully. Click here to check the status of the task.');
+                var link = ("/foreman_tasks/tasks/%taskId").
+                  replace('%taskId', response.task.id);
```

Comment:
Nitpicky thing but usually the period is on this line `.replace` vs `replace`. You could also make this one line since its not too long

Reply:
If we really want to enforce this, I'd be in favor of turning on [the eslint rule](https://eslint.org/docs/rules/dot-location).